cpp/daal/src/algorithms/linear_model/linear_model_hyperparameter_impl.h

```diff
@@ -41,8 +41,11 @@ namespace internal
  */
 enum HyperparameterId
 {
-    denseUpdateStepBlockSize = 0,
-    hyperparameterIdCount    = denseUpdateStepBlockSize + 1
+    denseUpdateStepBlockSize     = 0,
+    denseUpdateMaxColsBatched    = 1,
+    denseSmallRowsThreshold      = 2,
+    denseSmallRowsMaxColsBatched = 3,
+    hyperparameterIdCount        = 4
```

Comment:
It would be less changes when some new ID is added:
```suggestion
    denseUpdateStepBlockSize = 0,
    denseUpdateMaxColsBatched,
    denseSmallRowsThreshold,
    denseSmallRowsMaxColsBatched,
    hyperparameterIdCount = denseSmallRowsMaxColsBatched + 1
```

Reply:
@Vika-F But there's no guarantee that the numbers will be mapped in a sequence. In theory, the compiler would be free to map `denseSmallRowsMaxColsBatched` to '100' and then we'd end up with an incorrect count of parameters.

Reply:
I understand your concern. Yes, you are right in theory. Let's leave it like this.